validation/charts/appco/istio.go

```diff
@@ -7,34 +7,37 @@ import (
 	"github.com/rancher/shepherd/clients/rancher"
 	extencharts "github.com/rancher/shepherd/extensions/charts"
 	"github.com/rancher/shepherd/extensions/kubectl"
-	namegen "github.com/rancher/shepherd/pkg/namegenerator"
 	"github.com/rancher/tests/actions/charts"
-	kubeapiNamespaces "github.com/rancher/tests/actions/kubeapi/namespaces"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 const (
-	RancherIstioSecret string = "application-collection"
+	ExpectedDeployLog                                  = "deployed"
+	IstioCanaryRevisionApp                             = "istiod-canary"
+	RancherIstioSecretName                      string = `application-collection`
+	IstioAmbientModeSet                         string = `--set cni.enabled=true,ztunnel.enabled=true --set istiod.cni.enabled=false --set cni.profile=ambient,istiod.profile=ambient,ztunnel.profile=ambient`
```

Comment:
these don't need to be public afaik, since they are test-level they can just be private functions